### PR TITLE
docs(image): explain deprecation of category methods

### LINKED ISSUE #2019: Improve deprecation message for faker.image.abstract() etc
The deprecation message for `faker.image.abstract()` etc points to `faker.image.url()` which doesn't support a category. 

image

_Originally posted by @matthewmayer in https://github.com/faker-js/faker/issues/2014#issuecomment-1494386708_

Maybe we should change that to point to faker.image.urlLoremFlickr({category:"abstract"}) instead then, since only this provider supports categories?

Alternatively faker.image.url() could be modified to accept a category parameter (in which case it would always use LoremFlickr instead of Picsum or LoremFlickr alternately)

### PR DESCRIPTION
**Pull request by matthewmayer**
fix #2019

Not sure if an explanation is required next to the @see as well as in the deprecation section?

image

### CODE REVIEW

**Review by xDivisionByZerox — CHANGES_REQUESTED** [review]
Please update the runtime warning messages in the `proposed` properties as well.
Otherwise looks good.

File: src/modules/image/index.ts [hunk 1]
```diff
@@ -501,14 +501,18 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.abstract() // 'https://loremflickr.com/640/480/abstract'
    * faker.image.abstract(1234, 2345) // 'https://loremflickr.com/1234/2345/abstract'
    * faker.image.abstract(1234, 2345, true) // 'https://loremflickr.com/1234/2345/abstract?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'abstract'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'abstract' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

File: src/modules/image/index.ts [hunk 2]
```diff
@@ -528,14 +532,18 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.animals() // 'https://loremflickr.com/640/480/animals'
    * faker.image.animals(1234, 2345) // 'https://loremflickr.com/1234/2345/animals'
    * faker.image.animals(1234, 2345, true) // 'https://loremflickr.com/1234/2345/animals?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'animals'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'animals' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

File: src/modules/image/index.ts [hunk 3]
```diff
@@ -555,14 +563,19 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.business() // 'https://loremflickr.com/640/480/business'
    * faker.image.business(1234, 2345) // 'https://loremflickr.com/1234/2345/business'
    * faker.image.business(1234, 2345, true) // 'https://loremflickr.com/1234/2345/business?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'business'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'business' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

File: src/modules/image/index.ts [hunk 4]
```diff
@@ -582,14 +595,17 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.cats() // 'https://loremflickr.com/640/480/cats'
    * faker.image.cats(1234, 2345) // 'https://loremflickr.com/1234/2345/cats'
    * faker.image.cats(1234, 2345, true) // 'https://loremflickr.com/1234/2345/cats?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'cats'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'cats' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

File: src/modules/image/index.ts [hunk 5]
```diff
@@ -609,14 +625,17 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.city() // 'https://loremflickr.com/640/480/city'
    * faker.image.city(1234, 2345) // 'https://loremflickr.com/1234/2345/city'
    * faker.image.city(1234, 2345, true) // 'https://loremflickr.com/1234/2345/city?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'city'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'city' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

File: src/modules/image/index.ts [hunk 6]
```diff
@@ -636,14 +655,17 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.food() // 'https://loremflickr.com/640/480/food'
    * faker.image.food(1234, 2345) // 'https://loremflickr.com/1234/2345/food'
    * faker.image.food(1234, 2345, true) // 'https://loremflickr.com/1234/2345/food?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'food'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'food' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

File: src/modules/image/index.ts [hunk 7]
```diff
@@ -663,14 +685,17 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.nightlife() // 'https://loremflickr.com/640/480/nightlife'
    * faker.image.nightlife(1234, 2345) // 'https://loremflickr.com/1234/2345/nightlife'
    * faker.image.nightlife(1234, 2345, true) // 'https://loremflickr.com/1234/2345/nightlife?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'nightlife'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'nightlife' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

File: src/modules/image/index.ts [hunk 8]
```diff
@@ -690,14 +715,17 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.fashion() // 'https://loremflickr.com/640/480/fashion'
    * faker.image.fashion(1234, 2345) // 'https://loremflickr.com/1234/2345/fashion'
    * faker.image.fashion(1234, 2345, true) // 'https://loremflickr.com/1234/2345/fashion?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'fashion'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'fashion' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

File: src/modules/image/index.ts [hunk 9]
```diff
@@ -717,14 +745,17 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.people() // 'https://loremflickr.com/640/480/people'
    * faker.image.people(1234, 2345) // 'https://loremflickr.com/1234/2345/people'
    * faker.image.people(1234, 2345, true) // 'https://loremflickr.com/1234/2345/people?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'people'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'people' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

File: src/modules/image/index.ts [hunk 10]
```diff
@@ -744,14 +775,17 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.nature() // 'https://loremflickr.com/640/480/nature'
    * faker.image.nature(1234, 2345) // 'https://loremflickr.com/1234/2345/nature'
    * faker.image.nature(1234, 2345, true) // 'https://loremflickr.com/1234/2345/nature?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'nature'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'nature' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

File: src/modules/image/index.ts [hunk 11]
```diff
@@ -771,14 +805,17 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.sports() // 'https://loremflickr.com/640/480/sports'
    * faker.image.sports(1234, 2345) // 'https://loremflickr.com/1234/2345/sports'
    * faker.image.sports(1234, 2345, true) // 'https://loremflickr.com/1234/2345/sports?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'sports'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'sports' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

File: src/modules/image/index.ts [hunk 12]
```diff
@@ -798,14 +835,17 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.technics() // 'https://loremflickr.com/640/480/technics'
    * faker.image.technics(1234, 2345) // 'https://loremflickr.com/1234/2345/technics'
    * faker.image.technics(1234, 2345, true) // 'https://loremflickr.com/1234/2345/technics?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'technics'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'technics' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

File: src/modules/image/index.ts [hunk 13]
```diff
@@ -825,14 +865,19 @@ export class ImageModule {
...
+   *
    * @example
    * faker.image.transport() // 'https://loremflickr.com/640/480/transport'
    * faker.image.transport(1234, 2345) // 'https://loremflickr.com/1234/2345/transport'
    * faker.image.transport(1234, 2345, true) // 'https://loremflickr.com/1234/2345/transport?lock=56789'
    *
    * @since 2.0.1
    *
-   * @deprecated Use `faker.image.url` instead.
+   * @deprecated Use `faker.image.urlLoremFlickr({category: 'transport'})` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * @deprecated Use `faker.image.urlLoremFlickr({ category: 'transport' })` if you want an image from LoremFlickr in the correct category, or `faker.image.url()` if you just want any image.
```

**Review by xDivisionByZerox — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**